### ISSUE TITLE
date.php is not quarterly aware

### ISSUE THREAD

**Issue by dlangille** [FreshPorts/freshports#221]
Visit https://stagegit.freshports.org/?branch=quarterly

Scroll to bottom, click on Yesterday's commits (e.g. `https://stagegit.freshports.org/date.php?date=2021/3/21`)

* link does not include `branch=quarterly` 
* It you add it manually, it has no effect

**Comment by dlangille** [comment]
fixed via 27bd2030ce2151ee517c35e987dba938a944cad5

**Comment by dlangille** [comment]
Yes, `date.php` is now branch-aware. However, the link from the homepage will not propogate a `branch=` parameter.

**Comment by dlangille** [comment]
Reopened until this hits prod.

**Comment by grahamperrin** [comment]
Does this relate to the _2022Q4_ listings that are e.g. interspersed at the home page?

**Comment by dlangille** [comment]
> Does this relate to the _2022Q4_ listings that are e.g. interspersed at the home page?

By design, the home page shows both main and quarterly branch commits. Any quarterly branch.

It is not related to this issue from what I know.
